### PR TITLE
Added ng-content

### PR DESCRIPTION
**Pull request by thekiba**
Нужен перевод

### CODE REVIEW

File: snippets/ng-content.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+export class Wrapper {}
+```
+
+```html
+<wrapper>
+    <h1>Hello World!</h1>
+</wrapper>
+```
+
+# Bonus
```
Comment (kirjs) [inline snippets/ng-content.md]: Move this to interactive example

**Review by kirjs — CHANGES_REQUESTED** [review]

File: snippets/ng-content.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+level: beginner
+author: thekiba
+tags:
+  - good-to-know
+  - tips
+  - components
+---
+
+# Content
+With ng-content **ng-content** you can pass any elements to a component.
```
Comment (fetis) [inline snippets/ng-content.md]:
```suggestion
With  `ng-content` you can pass any elements to a component. 
```

File: snippets/ng-content.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+})
+export class CardHeader {}
+
+@Component({
+  selector: 'card-body',
+  template: template('card-body')
+})
+export class CardBody {}
+
+@Component({
```
Comment (fetis) [inline snippets/ng-content.md:67]: too many component in the example

**Review by fetis — CHANGES_REQUESTED** [review]
still, it's Angular basics from angular.io

**Review by kirjs — APPROVED** [review]

File: snippets/ng-content.md (hunk 3 above)
Comment (kirjs) [reply snippets/ng-content.md:67]: +1, I'd keep like 2, that would be enough to illustrate the example

**Review by kirjs — CHANGES_REQUESTED** [review]

**Review by NothingEverHappens — APPROVED** [review]

**Review by NothingEverHappens — APPROVED** [review]

**Review by NothingEverHappens — APPROVED** [review]

**Review by kirjs — APPROVED** [review]

**Review by fetis — APPROVED** [review]
if it helps...

**Review by kirjs — APPROVED** [review]

**Pull request merged**